### PR TITLE
Bump to v3.0.0-RC3

### PR DESCRIPTION
**Pull request by mheon**
As the title says

### CODE REVIEW

**Comment by mheon** [comment]
@containers/podman-maintainers PTAL and merge

File: version/version.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 // NOTE: remember to bump the version at the top
 // of the top-level README.md file when this is
 // bumped.
-var Version = semver.MustParse("3.0.0-rc3")
+var Version = semver.MustParse("3.0.0")
```
Comment (vrothberg) [inline version/version.go:11]: Shouldn't this go back to `-dev`?

Comment (mheon) [reply version/version.go:11]: You are extremely correct

**Review by vrothberg — COMMENTED** [review]
LGTM

**Review by saschagrunert — APPROVED** [review]
LGTM

**Pull request merged**